main.c

```diff
@@ -163,7 +169,7 @@ static void layer_surface_configure(void *data,
 	surface->width = width;
 	surface->height = height;
 	zwlr_layer_surface_v1_ack_configure(layer_surface, serial);
-	render_frame(surface);
+	render_frame_background(surface);
```

Comment:
We should probably adjust the subsurface position for the new size here.

Reply:
render_frame(surface) re-added.